packages/core/data-transfer/package.json

```diff
@@ -61,7 +61,7 @@
   "devDependencies": {
     "@strapi/database": "workspace:*",
     "@strapi/pack-up": "5.0.0",
-    "@types/fs-extra": "9.0.13",
+    "@types/fs-extra": "11.0.4",
```

Comment:
Other packages use fs-extra but don't have the @types dependency, should we add it?

Reply:
Yeah good idea I'll add those